include/oxli/hashtable.hh

```diff
@@ -140,6 +140,7 @@ protected:
     HashIntoType    bitmask;
     unsigned int    _nbits_sub_1;
 
+public:
```

Comment:
What is a better way of doing this? I needed to make the destructor of `Hashtable` public for unique_ptr's `reset()` to work. Could someone tell me why it was protected? I assume it is to make sure it doesn't get called for a derived class. Wisdom welcome.